### ISSUE TITLE
Finalize FAQs page and components

### ISSUE THREAD

**Issue by BastiDood** [up-csi/up-csi.github.io#21]
* [ ] up-csi/engg-task-tracker#3

**Comment by ehrelevant** [comment]
I'm not particularly sure what the plans for this originally were but, as of the moment, there is no FAQ section on the projects page of the website nor existing designs for such a section. I also don't know how much progress has been made in preparing the needed details for the FAQs section (contact information, portfolio resources, previous experience, etc.) besides that it wasn't completed.

**Comment by VeeIsForVanana** [comment]
It seems @BastiDood 's original intent with this was a one-stop-shop for clients to know the services we offer and how to contact us. This, I think isn't just an FAQ page but a kind-of Contact Us page. I'll need to coordinate with Exte on the details that are appropriate for this.

TODO:

- [ ] Talk to Exte re: details to put on this page
- [ ] Complete portfolio
- [ ] Create mock-ups

**Comment by BastiDood** [comment]
> It seems @BastiDood 's original intent with this was a one-stop-shop for clients to know the services we offer and how to contact us. This, I think isn't just an FAQ page but a kind-of Contact Us page.

To add more context here, since we're dealing with `adapter-static`, we don't actually have a back-end web server running the website; everything is compiled down to HTML.

When we say "Contact Us", this should be a list of social media cards instead of an email form (as is typical in many marketing sites). Why? An email form requires third-party integrations: the simplest being the Gmail API via OAuth 2.0— something @VeeIsForVanana and I have worked with in the [DRAP](https://github.com/BastiDood/drap) project.

Now, if you're up for the challenge to use email contact forms, don't let me stop you. Here are a few things to consider, though:

* Deploy the website to an actual hosting provider that lets us run `adapter-node` web servers.
* Choose the email integration. As mentioned earlier, the Gmail API is your best bet.
* Make sure to rate-limit the email submission to mitigate DDoS attacks and general rate limit exhaustion.
* Handle the API secrets properly. These will have to be passed down to future directors of engineering from this point forward.

**Comment by VeeIsForVanana** [comment]
Yeah I'm more favorable to the social media cards idea. It should probably be a page featuring the organization's contacts, along with answering some FAQs on our projects pipeline. In any case, thanks for the input!
